.github/workflows/ci.yml

```diff
@@ -10,7 +14,7 @@ jobs:
           - otp: 25.1
             elixir: 1.15.4
           - otp: 21.3
-            elixir: 1.7.4
+            elixir: 1.11.4
```

Comment:
I have my doubts about this change, as we are no longer providing backwards compatibility with older versions of Elixir. But I think we can do that because if someone is using an Elixir version less than 1.11, they're going to have a lot of problems with the ecosystem. It will also be quite difficult for us to support older versions.

Reply:
Also on a related note, let's update the supported version of the Elixir in the `mix.exs`
https://github.com/elixirmoney/money/blob/c2d74142a334e6fd1f369b6018b97bb563a660a0/mix.exs#L13

Reply:
If we want to update dependencies we need to drop support for old versions of Elixir...

Reply:
If we don't test them on CI, it's equivalent to drop. I think that's fine, people with an old version of Elixir won't be able to upgrade, but we won't break their code